content/numpy/concepts/built-in-functions/terms/reshape/reshape.md

```diff
@@ -58,3 +58,43 @@ This produces the following output:
  [5]
  [6]]
 ```
+
+Optionally, a third parameter can be added to the syntax: `order`.
+
+`order` reads the elements from `array` in the current index order and places the elements into the reshaped array using the same index order. It can be set to `'C'`, `'F'`, or `'A'`. `C` means to read/write the elements using C-like index order, or in other words, the elements are placed row by row. The last axis index (columns) changes first (fastest), and the first axis index (rows) changes afterward (slowest). `F` means to read/write the elements using Fortran-like index order, where the elements are placed column by column. Here, the first axis index (rows) changes faster while the last axis index (columns) changes slower. Using `C` or `F` in `.reshape()` means that the function is only reshaping the way that indexes are placed in the array, and not like data physically stored in the computer memory (memory layout). `A` means to read/write the elements in Fortran-like index order if `array` is Fortran contiguous in memory, meaning, in a sequencial and continuous manner without gaps between elements, or in C-like index order otherwise.
+
+```pseudo
+numpy.reshape(array, newshape, order = 'C')
+```
```

Comment:
This can be edited in the ##Syntax section itself. @NelsonSanti What we can do is, go to the ## Syntax section above, modify that syntax by adding the `order` parameter as well. Then after the syntax, use bullet points to address the parameters. Example:

```pseudo
numpy.reshape(array, newshape, order = 'C')
```
- `array`: Describe this
- `newshape`: describe this
- `order`: Describe this
  - `C`: describe 
  - `F`: describe
  - `A`: descibe

Reply:
This way a proper structure can be maintained